### ISSUE TITLE
use pystac-client instead of sat-search

### ISSUE THREAD

**Issue by rsignell-usgs** [TomAugspurger/landcover-demo-gtc-2021#1]
@matthewhanson says `sat-search` is deprecated in favor of https://github.com/stac-utils/pystac-client

**Comment by TomAugspurger** [comment]
Indeed. That's what we use in https://planetarycomputer.microsoft.com/docs/tutorials/landcover/.

```python
import pystac_client

api = pystac_client.Client.open(
    "https://planetarycomputer-staging.microsoft.com/api/stac/v1/"
)
search_2013 = api.search(
    intersects=area_of_interest,
    datetime="2012-12-31T00:00:00Z/2014-01-01T00:00:00Z",
    limit=500,
    collections=["naip"]
)

search_2017 = api.search(
    intersects=area_of_interest,
    datetime="2016-12-31T00:00:00Z/2018-01-01T00:00:00Z",
    limit=500,
    collections=["naip"]
)

print(search_2013.matched())
print(search_2017.matched())
```

I just added a note to the readme saying that https://planetarycomputer.microsoft.com/docs/tutorials/landcover/ has an up-to-date version.

It'd be nice if there were a way to download the notebook from that page. I'll see if that's doable.

**Comment by rsignell-usgs** [comment]
cc @ocefpaf

**Comment by TomAugspurger** [comment]
FYI Filipe, if you're looking at the environment / binder stuff, I just pushed an update to the `binder` branch: https://github.com/TomAugspurger/landcover-demo-gtc-2021/commit/129b39f85f4ed8455b4ecae8f483a4e2cf07fd74.

That's what I'm using for https://planetarycomputer.microsoft.com/docs/tutorials/landcover/, and things seem to work OK.

**Comment by ocefpaf** [comment]
Thanks @TomAugspurger! I'm trying to make it work with less channels and less pip install calls. Not there yet but getting close to it. (Fixing a bunch of conda-forge pkgs on the way.)

**Comment by TomAugspurger** [comment]
Great! I'm about to submit PRs to staged-recipes for planetarycomputer and pystac-client.

**Comment by TomAugspurger** [comment]
Oh, and I completely forgot about https://github.com/conda-forge/segmentation-models-pytorch-feedstock/pull/8. Whoops.
